openedx/core/djangoapps/programs/models.py

```diff
@@ -25,23 +21,6 @@ class ProgramsApiConfig(ConfigurationModel):
     internal_service_url = models.URLField(verbose_name=_("Internal Service URL"))
     public_service_url = models.URLField(verbose_name=_("Public Service URL"))
 
-    authoring_app_js_path = models.CharField(
```

Comment:
Please move these changes to a separate PR, to be merged later. Model fields should only be removed after the code which depends on them has been updated. This is because migrations are run before any code is updated. If you remove these fields now, the code currently deployed which depends on these fields will break, and it will make rolling back difficult, too.